wies/core/static/css/rvo_overwrite.css

```diff
@@ -2,6 +2,42 @@
   width: 100%;
 }
 
+/* Sidepanel data list styling */
+.rvo-dialog .rvo-data-list {
```

Comment:
most of this CSS seems specific for the sidepanel. by placing it in the generic rvo_overwrite it becomes active for all dialogues we will add. i dont think that's desirable

also it contains non-rvo classes. we also have `base.css` for the classes that apply to all code. if it should only be applied to a specific page/part you should create a separate file for it